web/src/popover_menus.js

```diff
@@ -203,11 +203,37 @@ export function initialize() {
             }
 
             instance.setContent(parse_html(render_left_sidebar_stream_setting_popover()));
+            //  When showing the popover menu, we want the
+            // "Add streams" and the "Filter streams" tooltip
+            //  to appear below the "Add streams" icon.
+            const add_streams_tooltip = $("#add_streams_tooltip").get(0);
+            add_streams_tooltip._tippy?.setProps({
+                placement: "bottom",
+            });
+            const filter_streams_tooltip = $("#filter_streams_tooltip").get(0);
+            // If `filter_streams_tooltip` is not triggered yet, this will set its initial placement.
+            filter_streams_tooltip.dataset.tippyPlacement = "bottom";
+            filter_streams_tooltip._tippy?.setProps({
+                placement: "bottom",
+            });
             return true;
         },
         onHidden(instance) {
             instance.destroy();
             popover_instances.stream_settings = undefined;
+            //  After the popover menu is closed, we want the
+            //  "Add streams" and the "Filter streams" tooltip
+            //  to appear at it's original position that is
+            //  above the "Add streams" icon.
+            const add_streams_tooltip = $("#add_streams_tooltip").get(0);
+            add_streams_tooltip._tippy?.setProps({
+                placement: "top",
+            });
+            const filter_streams_tooltip = $("#filter_streams_tooltip").get(0);
+            filter_streams_tooltip.dataset.tippyPlacement = "top";
+            filter_streams_tooltip._tippy?.setProps({
+                placement: "top",
+            });
```

Comment:
I changed this a bit and added fallback in case `filter_streams_tooltip` is not triggered yet by the user. This cannot be the case with `add_streams_tooltip` since it will be triggered when user clicks on the button to open the popover.

Reply:
Yeah you are right when the user hasn't triggered the `filter_streams_tooltip` then the `setProps` function has no effect. This is also clear in manual testing.

<details>
<summary>Before Opening Tippy </summary>
![Before Opening Tippy](https://user-images.githubusercontent.com/35286603/230146964-d58d3cb0-70ce-4eca-bbde-09487c3f6b60.gif)
</details>

<details>
<summary>After Opening Tippy </summary>
![After Opening Tippy](https://user-images.githubusercontent.com/35286603/230146937-65e9a605-0bdc-4415-9414-ea94ea1e8e02.gif)
</details>

But now it is fixed, thanks :heart: